Source/Runtime/Session Management Client.lvproj

```diff
@@ -66,6 +66,7 @@
 				<Item Name="LVBoundsTypeDef.ctl" Type="VI" URL="/&lt;vilib&gt;/Utility/miscctls.llb/LVBoundsTypeDef.ctl"/>
 				<Item Name="BuildHelpPath.vi" Type="VI" URL="/&lt;vilib&gt;/Utility/error.llb/BuildHelpPath.vi"/>
 				<Item Name="GetHelpDir.vi" Type="VI" URL="/&lt;vilib&gt;/Utility/error.llb/GetHelpDir.vi"/>
+				<Item Name="OneOfTemplate.lvclass" Type="LVClass" URL="/&lt;vilib&gt;/gRPC/gRPC Server and Client Template [2]/Oneof Template/OneOfTemplate.lvclass"/>
```

Comment:
I don't think the runtime code should be dependent on the 'gRPC Server and Client Template' which is used to generate the stubs. Do you know why this is added or needed? It looks like it's the first reference to that.

Reply:
@dixonjoel 
You are correct. The Template is not a dependency. However, when we generated stubs using this vi, I guess its dependency class got added as part of the session management project file at the end of vi execution somehow as it is in memory along with the session management project. I guess I should have closed the `gRPC Server and Client Template [2]\gRPC Scripting Tools\Main.vi` and saved the project file.

But the next time opening the project in LabVIEW automatically removes the 'OneOfTemplate.lvclass' name in the project file since it is not linked to any other files or projects. Anyway, I opened and saved this project file again and pushed this change without the unnecessary reference.

Thanks for pointing out the unnecessary reference in this project.